checks/net.go

```diff
@@ -260,3 +262,16 @@ func connectionPIDs(conns []ebpf.ConnectionStats) []uint32 {
 	}
 	return pids
 }
+
+func connectionPIDs(conns []*model.Connection) []int32 {
```

Comment:
This duplication is odd. Can we just simplify it and have a single function? Or just reuse the previous function?

Reply:
That's kinda hard. One takes `ebpf.ConnectionStats` as input and the other takes `*model.Connection`. I can convert both input to be `[]interface` but every time we use this function we have to convert the input into `[]interface`.